### PR TITLE
add mxnet install script for fedora python

### PR DESCRIPTION
**Pull request by sethbrin**
Add install script of mxnet python  for fedora machine, it tested for the Fedora 21.0

### CODE REVIEW

**Comment by zihaolucky** [comment]
What if user install MXNet in anaconda's env? I mean the required packages might also be installed under this env, where `sudo` is not recommended.

**Comment by sethbrin** [comment]
I just write the script as install-mxnet-ubuntu-python.sh, when the user use anaconda or some other virtual environment, the user should rewrite the script.

File: setup-utils/install-mxnet-fedora-python.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+make -j$(nproc)
+
+echo "Installing Numpy..."
+sudo yum install numpy
+
+echo "Installing Python setuptools..."
+sudo yum install -y python-setuptools python-pip
+
+echo "Installing Python package for MXNet..."
+cd python; sudo python setup.py install
```
Comment (piiswrong) [inline setup-utils/install-mxnet-fedora-python.sh]: you can skip this. having PYTHONPATH is enough

Comment (sethbrin) [inline setup-utils/install-mxnet-fedora-python.sh]: fixed

Comment (piiswrong) [inline setup-utils/install-mxnet-fedora-python.sh]: no. I mean remove `cd python; sudo python setup.py install` and keep export PYTHONPATH...

**Pull request merged**